pkg/quay/quay.go

```diff
@@ -509,20 +510,13 @@ func (c *QuayClient) GetAllRobotAccounts(organization string) ([]RobotAccount, e
 // e.g. `org+robot` will be changed to `robot`, `robot` will stay `robot`
 func handleRobotName(robotName string) (string, error) {
 	// Regexp from quay api `^([a-z0-9]+(?:[._-][a-z0-9]+)*)$` with one plus sign in the middle allowed (representing longname)
-	r, err := regexp.Compile(`^[a-z0-9]+(?:[._-][a-z0-9]+)*(?:\+[a-z0-9]+(?:[._-][a-z0-9]+)*)?$`)
+	r, _ := regexp.Compile(`^[a-z0-9]+(?:[._-][a-z0-9]+)*(?:\+[a-z0-9]+(?:[._-][a-z0-9]+)*)?$`)
 	robotName = strings.TrimSpace(robotName)
-	if err != nil {
```

Comment:
we now it compiles now. Is there not a chance that we would update this in the future?